config-templates/ibm/ibm-vpc-block-csi-driver/5.1/metadata.json

```diff
@@ -43,7 +43,7 @@
     },
     {
       "name": "supported-ocp-versions",
-      "value": "4.10, 4.11, 4.12, 4.13, 4.14"
+      "value": "4.10, 4.11, 4.12, 4.13, 4.14, 4.15"
```

Comment:
Let us remove `4.10, 4.11` since these versions are not supported in IBM Cloud anymore.